src/apps/fee-list/FeeList.tsx

```diff
@@ -76,15 +97,32 @@ const FeeList: FC = () => {
             />
           </>
         )}
-        <List
-          dataCy="fee-list"
-          listHeader={t("unpaidFeesFirstHeadlineText")}
-          openDetailsModalClickEvent={openDetailsModalClickEvent}
-          fees={fbsFees}
-          totalText={t("totalText", {
-            placeholders: { "@total": totalFeeAmount.toString() }
-          })}
-        />
+        {getFeesBasedOnPayableByClient(fbsFees, true).length > 0 && (
```

Comment:
Nopes, because the function uses an [array.prototype.filter()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter) which returns an empty array in case no array items match the filter criteria. And google says that arrays, even empty, are always evaluated as true in JS.